### DISCUSSION TITLE
How do I access MDN offline?

### DISCUSSION THREAD

**Discussion by keshane** [mdn/mdn-community#712] [MDN Plus] [answered]
I've tried to browse the MDN docs offline, but don't understand how it's supposed to happen. Am I supposed to visit MDN like normal and be able to browse the docs?

I have supposedly downloaded the docs, but when I turn off my laptop's internet connection and try to visit the site, it says I'm offline. See attached screenshots.
![Screenshot from 2024-07-27 14-36-46](https://github.com/user-attachments/assets/fedf308d-1d42-412c-b9fb-e601f7a6031e)
![Screenshot from 2024-07-27 14-37-34](https://github.com/user-attachments/assets/e3abe15b-1792-4e77-a24c-0949eab43348)

Is it the service worker that's supposed to to serve the offline files? From multiple trials, it seems like the service worker stays registered for the first couple of pages as I click on links, but then disappears after a few clicks.

**Comment by pransh15** [comment, marked as answer]
Hey @keshane 👋 
You are doing everything right, as far as I can see. I suggest that you [clear stored data](https://developer.mozilla.org/en-US/plus/docs/features/offline#clear-stored-data), and try again. That usually helps me out.

One thing to note is that MDN Offline does not work on Incognito/Private Mode, if in case you are trying to access MDN Offline that way.

**Reply by keshane** [reply]
Thanks! I cleared the stored data and redownloaded everything before a recent cross-country plane ride. That seems to have worked this time.

**Comment by pransh15** [comment]
Closing this as resolved. ✅
